vector/src/main/res/layout/view_room_detail_toolbar.xml

```diff
@@ -21,26 +21,26 @@
 
     <im.vector.app.core.ui.views.ShieldImageView
         android:id="@+id/roomToolbarDecorationImageView"
-        android:layout_width="17dp"
-        android:layout_height="17dp"
-        android:layout_marginStart="5dp"
-        android:layout_marginTop="2dp"
-        app:layout_constraintBottom_toBottomOf="@id/roomToolbarTitleView"
-        app:layout_constraintStart_toEndOf="@id/roomToolbarAvatarImageView"
-        app:layout_constraintTop_toTopOf="@id/roomToolbarTitleView" />
+        android:layout_width="11dp"
+        android:layout_height="13dp"
+        tools:ignore="MissingConstraints"
```

Comment:
Please format the file. `tools` attribute has to be the latest attribute. You can type Maj+Option+Cmd+L in Android Studio to check the box to rearrange code.